src/device-api/actions.ts

```diff
@@ -151,9 +151,6 @@ export const doPurge = async (appId: number, force: boolean = false) => {
 			keepVolumes: false,
 			force,
 		});
-		// Restore user app after purge
-		currentState.local.apps[appId] = app;
-		await deviceState.applyIntermediateTarget(currentState);
 		logger.logSystemMessage('Purged data', { appId }, 'Purge data success');
```

Comment:
Optional change, but the message wouldn't technically be accurate as a purge has not yet occurred yeah?

Reply:
The purge happens when the `applyIntermediate` call on 150 finishes, because that's what removes the app. The call on finalize re-installs the app